src/mbgl/gl/context.cpp

```diff
@@ -37,10 +38,16 @@ static_assert(underlying_type(TextureFormat::Alpha) == GL_ALPHA, "OpenGL type mi
 Context::~Context() {
     reset();
 }
+    
+void Context::logGlMaxVertexAttribsValue() {
+    int glMaxVertexAttribs;
+    glGetIntegerv(GL_MAX_VERTEX_ATTRIBS, &glMaxVertexAttribs);
```

Comment:
Is it possible to wrap this call under MBGL_CHECK_ERROR macro. Its helpful to catch GL bugs. Its stripped out in release builds.